src/commands/game_commands/play.ts

```diff
@@ -233,47 +262,158 @@ export default class PlayCommand implements BaseCommand {
+        {
+            name: PlaySlashCommands[GameType.CLASSIC],
+            description: LocalizationManager.localizer.translate(
+                LocaleType.EN,
+                "command.play.help.example.classic"
+            ),
+            type: Eris.Constants.ApplicationCommandTypes.CHAT_INPUT,
+        },
+        {
+            name: PlaySlashCommands[GameType.TEAMS],
+            description: LocalizationManager.localizer.translate(
+                LocaleType.EN,
+                "command.play.help.example.teams"
+            ),
+            type: Eris.Constants.ApplicationCommandTypes.CHAT_INPUT,
+        },
+        {
+            name: PlaySlashCommands[GameType.ELIMINATION],
+            description: LocalizationManager.localizer.translate(
+                LocaleType.EN,
+                "command.play.help.example.elimination"
+            ),
+            type: Eris.Constants.ApplicationCommandTypes.CHAT_INPUT,
+            options: [
+                {
+                    name: "lives",
+                    description: LocalizationManager.localizer.translate(
+                        LocaleType.EN,
+                        "command.play.help.interaction.lives",
+                        {
+                            lives: `\`${ELIMINATION_DEFAULT_LIVES}\``,
+                        }
+                    ),
+                    type: Eris.Constants.ApplicationCommandOptionTypes.INTEGER,
+                },
+            ],
+        },
+    ];
+
+    static async processChatInputInteraction(
+        interaction: Eris.CommandInteraction,
+        messageContext: MessageContext
+    ): Promise<void> {
+        const SLASH_COMMAND_TO_GAME_TYPE = {
+            [PlaySlashCommands[GameType.CLASSIC]]: GameType.CLASSIC,
+            [PlaySlashCommands[GameType.ELIMINATION]]: GameType.ELIMINATION,
+            [PlaySlashCommands[GameType.TEAMS]]: GameType.TEAMS,
+        };
+
+        const gameType = SLASH_COMMAND_TO_GAME_TYPE[interaction.data.name];
+        let lives: number;
+        if (gameType === GameType.ELIMINATION) {
+            if (!interaction.data.options) {
+                lives = ELIMINATION_DEFAULT_LIVES;
+            } else {
+                lives = interaction.data.options[0]["value"] as number;
+                if (
+                    lives < ELIMINATION_MIN_LIVES ||
+                    lives > ELIMINATION_MAX_LIVES
+                ) {
+                    lives = ELIMINATION_DEFAULT_LIVES;
+                }
+            }
+        }
+
+        await PlayCommand.startGame(
+            messageContext,
+            gameType,
+            lives,
+            interaction
+        );
+    }
+
+    call = async ({ message, parsedMessage }: CommandArgs): Promise<void> => {
+        const gameType =
+            (parsedMessage.components[0]?.toLowerCase() as GameType) ??
+            GameType.CLASSIC;
+
+        let lives: number;
+        if (gameType === GameType.ELIMINATION) {
```

Comment:
Can we re-use this `lives` logic for both slash command and text input cases into `PlayCommand.startGame`?